prow/getting_started_deploy.md

```diff
@@ -280,6 +280,49 @@ rolebindings afterwards.
 fields after deploying the prow components, you will need to redeploy them
 so that they pick up the change.
 
+### Configure Cloud Storage
```

Comment:
Nice! Ideally I'd vote for a `bootstrap-gcs.sh` script people can run which guides them through this process.

Reply:
Could be something to add to `tackle`? Or similar if we have automated getting started tool